types/lodash/common/object.d.ts

```diff
@@ -3173,10 +3242,11 @@ declare module "../index" {
     }
 
     interface LoDashImplicitWrapper<TValue> {
+        // NOTE: this overload also works for NumericDictionary and arbitrary object types.
```

Comment:
Is this note really necessary? If it is, I would put it on the `LoDashStatic` version, not here.

Reply:
Oops. That comment is leftover from a different attempt at the signature.  Deleting the comment.